### DISCUSSION TITLE
Cart not updating in local development

### DISCUSSION THREAD

**Discussion by p6gb** [EvgeniyMukhamedjanov/liquid-ajax-cart#16] [Q&A]
I have just spent too much time looking for an error because the was not updating when products were added or removed in local development. I assumed this had to do with `liquid-ajax-cart` but then found out it's a known issue for local development and the _Dawn_ theme (see https://github.com/Shopify/dawn/issues/968).

Maybe this could be stated somewhere to keep others from making the same mistake.

**Comment by EvgeniyMukhamedjanov** [comment]
Hey @p6gb 
Thank you for the note. I'm thinking a lot about creation a manual how to integrate Liquid Ajax Cart with Dawn.
Liquid Ajax Cart were originally designed to build Ajax carts when you are creating a new theme from scratch. The issue with Dawn is that it has its own ajax-cart related scripts: adding products to cart, displaying quantity, plus, minus buttons on the cart page etc. Technically when you set Liquid Ajax Cart on top of Dawn theme -- you have two different scripts that provide Ajax Cart functionality and it might cause bugs. 

To make this manual I need to know what functionality you use from Liquid Ajax Cart when you set it up on top of Dawn. Do you only use Liquid Ajax Cart to build the side-cart or you also rebuild the cart page? Do you use any of Liquid Ajax Cart JavaScript functions or only data-attributes ?

**Reply by EvgeniyMukhamedjanov** [reply]
I also had an idea to create my own fork of Dawn where all the ajax-cart scripts would be removed and replaced with Liquid Ajax Cart. But when I started to work on it I realized that it is way more difficult and time consuming. So still thinking about the solution :-)

**Reply by EvgeniyMukhamedjanov** [reply]
I even created a suggestion here
https://github.com/Shopify/dawn/issues/1319
but still no response.

**Reply by p6gb** [reply]
I am basically just setting it on top of Dawn functionality and hoping that everything goes well ¯_༼ᴼل͜ᴼ༽_/¯ 
I would also like to use Liquid Ajax Cart to rebuild the cart page but have not done that yet. Do you have recommendations on how to do that?
I also would like to see an easy way to replace the default cart functionality with Liquid Ajax Cart. Your suggestion looks good, I upvoted it. 

Many thanks for your work, it helped me a lot!

**Reply by EvgeniyMukhamedjanov** [reply]
@p6gb 
Thanks for the upvote!
Rebuilding the cart page is not difficult. It is the easiest part. I'll install the fresh Dawn tomorrow and check if they didn't update anything. After the checking I'll guide you what to change to replace functionality with Liquid Ajax Cart.
